OfficeDev/office-ui-fabric-react
# Only update ExtendedPicker state on input events that are not mid-composition

#### Pull request checklist

- ~~[ ] Addresses an existing issue: Fixes #0000~~
- [x] Include a change request file using `$ yarn change`

#### Description of changes

This is for a bug coming from outlook.

when composing input to the extendedpicker, we react to inputChanged events in order to notify the floating picker of its new search string, and query for new results. However, that doesn't align with the intent of a change coming from mid-IME composition. Pickers should not search with mid-composing inputs, since those change under the composition box & are usually not relevant until composed.

#### Focus areas to test
Autofill:
- run the test suite

ExtendedPickers.
Test plan:
1. run `yarn start` in `packages/office-ui-fabric-react`
2. confirm existing behaviour for non-composing input is the same(example: english language)
3. Switch to a composing input (e.g. hiragana jp)
4. compose a search
5. observe that, while still composing, the search results do not update,
6. complete the composition
7. observe the updated search results.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11303)
